Algorithm/QCAlgorithm.cs

```diff
@@ -297,6 +297,11 @@ public AlgorithmSettings Settings
         public IOptionChainProvider OptionChainProvider { get; private set; }
 
         /// <summary>
+        /// Gets the default order properties
+        /// </summary>
+        public IOrderProperties DefaultOrderProperties { get; set; }
```

Comment:
Out of curiosity, what would I have to go through in order to *not* use these default properties? Say for example, I'd like to issue this trade with different algo parameters or a different fa method?

Reply:
If you mean sending an order with different properties from those in `DefaultOrderProperties`, I believe we would need to have new overloads for Order, Buy, Sell and SetHoldings methods.
Maybe better address this in a separate PR ?

Reply:
```
DefaultOrderProperties = new RiskyGroupOrderProperties();
MarketOrder("SPY", 100);

DefaultOrderProperties = new SafeGroupOrderProperties();
MarketOrder("SPY", 100);
```

Reply:
Definitely agreed not in the PR, but suspect it will be something that comes up